server/src/test/java/org/elasticsearch/gateway/PersistedClusterStateServiceTests.java

```diff
@@ -55,21 +64,25 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicLong;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import static org.hamcrest.Matchers.allOf;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.lessThan;
 import static org.hamcrest.Matchers.nullValue;
 
-public class PersistedClusterStateServiceTests extends ESTestCase {
+public class PersistedClusterStateServiceTests extends ESAllocationTestCase {
```

Comment:
Seems wrong, this isn't an allocation test? I think we only need this for a single call to `newNode` -- let's do that inline instead (or move `newNode` somewhere more appropriate).